backend/src/cusco/sources/prr.py

```
@@ -0,0 +1,365 @@
+            return []
+
+        async with self._client() as client:
+            logger.info(f"Downloading PRR entidades from {url}...")
+            resp = await client.get(url)
+            resp.raise_for_status()
+            rows = self._parse_prr_entidades_xlsx(resp.content)
+            logger.info(f"Loaded {len(rows)} PRR entidades rows")
+            return rows
+
+    # Columns we actually read downstream — parser drops everything else to
+    # minimize row size (750K rows × unused columns = ~hundreds of MB).
```

Ruff (0.15.10)

[warning] Comment contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF003)